.github/workflows/subsystem-benchmarks.yml

```diff
@@ -57,26 +51,57 @@ jobs:
           rustup +nightly show
 
       - name: Run Benchmarks
-        continue-on-error: true
         id: run-benchmarks
         run: |
           cargo bench -p ${{ matrix.features.name }} --bench ${{ matrix.features.bench }} --features subsystem-benchmarks || echo "Benchmarks failed"
```

Comment:
```suggestion
          forklift cargo bench -p ${{ matrix.features.name }} --bench ${{ matrix.features.bench }} --features subsystem-benchmarks || echo "Benchmarks failed"
```

Reply:
@alvicsam applied all recommendations and updated the system benchmarks publishing to use environment scope + sequenced push to gh-pages 1by1 from artifacts